dao/pom.xml

```diff
@@ -194,6 +194,10 @@
             <groupId>org.elasticsearch.client</groupId>
             <artifactId>rest</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.persistence</groupId>
+            <artifactId>eclipselink</artifactId>
```

Comment:
I believe adding new persistence framework is wrong. Please remove this. You use it only in one place.

Reply:
Yes, I agree. Already removed.